### PR TITLE
feat: sambanova's langchain integration usage in entreprise knowledge retriever

### PR DESCRIPTION
**Pull request by snova-jorgep**
usage of sambanova's langchain integration
fix bug with sambanova embeddings

### CODE REVIEW

**Comment by snova-jorgep** [comment]
@snova-kwasia

File: enterprise_knowledge_retriever/src/document_retrieval.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def get_qa_retrieval_chain(self, vectorstore):
         """
         Generate a qa_retrieval chain using a language model.
 
-        This function uses a language model, specifically a SambaNovaEndpoint, to generate a qa_retrieval chain
+        This function uses a language model, specifically a SambaNova llm, to generate a qa_retrieval chain
```
Comment (snova-kwasia) [inline enterprise_knowledge_retriever/src/document_retrieval.py:221]: Capitalize to LLM please.

**Review by snova-kwasia — CHANGES_REQUESTED** [review]
One minor change re punctuation. Aside from that, all good.

File: enterprise_knowledge_retriever/src/document_retrieval.py (hunk 1 above)
Comment (snova-jorgep) [reply enterprise_knowledge_retriever/src/document_retrieval.py]: Done

**Review by snova-kwasia — APPROVED** [review]
Approved.

**Comment by snova-jorgep** [comment]
@snova-jamesv could you please merge this PR

**Pull request merged**